cmd/orchestration/app/options.go

```diff
@@ -41,7 +41,7 @@ func readAndValidateOptions(configFile string) (*Options, error) {
 	}
 
 	var opts Options
-	if err := yaml.Unmarshal(doc, &opts); err != nil {
+	if err := yaml.UnmarshalStrict(doc, &opts); err != nil {
```

Comment:
Belated question about this change - during our deployment process, do we roll back configmap changes if the deployment fails? If not, wouldn't this change potentially cause problem during failed deployments? Scenario being that we accidentally roll out bad config - at that point, not only will new versions of the app not start, but if any pods get rescheduled, they also won't start. We'd need to consciously roll back to the previous config/configmap if that doesn't happen automatically